packages/dd-trace/src/config.js

```diff
@@ -132,6 +132,11 @@ class Config {
       process.env.DD_TRACE_EXPERIMENTAL_INTERNAL_ERRORS_ENABLED,
       false
     )
+    const DD_APPSEC_ENABLED = coalesce(
+      options.experimental && options.experimental.appsec,
+      process.env.DD_EXPERIMENTAL_APPSEC_ENABLED,
```

Comment:
Did you want to also add the cross-language environment variable as a fallback as discussed?

Reply:
Talked about it with manager, and he doesn't like the idea. We'll settle with the `EXPERIMENTAL` syntax 👍